AUTHORS

```diff
@@ -62,4 +62,6 @@ Kieran Kaempen <kaempen2@illinois.edu>
 Clouds Flowing <clouds.flowing@gmail.com>
 Sylvain Corlay <sylvain.corlay@gmail.com>
 Alan Hu <alanlh2@illinois.edu>
-
+Rajiv Ranjan Singh <rajivperfect007@gmail.com>
+Francesco Bonazzi <franz.bonazzi@gmail.com>
+Simon Stelter <stelter@tzi.de>
```

Comment:
@ichumuh, @Upabjojr, @iamrajiv, let us know if you want to change your name or email here.

Reply:
wow thanks for that! I'd prefer simon.stelter@gmail.com as my email.